cmd/moco-backup/cmd/root.go

```diff
@@ -63,6 +64,17 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	defer func() {
+		err := recover()
+		if err != nil && errors.Is(err.(error), backup.ErrBadConnection) {
```

Comment:
if not, the error needs to be rethrown with panic.  Here's an example.
https://cs.opensource.google/go/go/+/refs/tags/go1.16.6:src/encoding/json/encode.go;l=322